.github/workflows/ci.yml

```
@@ -45,8 +45,11 @@ jobs:
       run: mix format --check-formatted
       if: ${{ matrix.lint }}
 
+    - name: Install dependencies
+      run: mix deps.get
```

```suggestion
    - name: Install dependencies
      run: mix deps.get --only test
```

Let's also move this above the format step. @wojtekmach has been updating CI for a lot of Elixir packages and has a nice template we can steal :) here's an example: dashbitco/nimble_parsec#87

---

Fixed.